alerting/provider/custom/custom.go

```diff
@@ -33,10 +41,28 @@ func (provider *AlertProvider) ToCustomAlertProvider(service *core.Service, aler
 	return provider
 }
 
+// GetPlaceholderValue returns the Placeholder value
+func (provider *AlertProvider) GetPlaceholderValue(name, status string) string {
```

Comment:
Since `ALERT_TRIGGERED_OR_RESOLVED` is the only placeholder which has one value or another, I believe that it's the only one which has a configuration that will be used like that.

As such, could you rename this function to something like `GetAlertStatePlaceholderValue(resolved bool)` which would use hardcoded values only inside that function instead of outside (e.g. `provider.Placeholders["ALERT_TRIGGERED_OR_RESOLVED"]["resolved"]`)?

That way, instead of calling `provider.GetPlaceholderValue("ALERT_TRIGGERED_OR_RESOLVED", "resolved")`, it'd be `provider.GetAlertStatePlaceholderValue(true)`.

Reply:
Hi @TwinProduction  => I just did the changes for you :)